torch/ao/quantization/qconfig.py

```diff
@@ -170,6 +172,12 @@ def __new__(cls, activation=torch.nn.Identity, weight=torch.nn.Identity):
 Fused version of `default_qat_config`, has performance benefits.
 """
 
+default_reuse_input_qconfig = QConfig(activation=default_reuse_input_observer,
+                                      weight=default_reuse_input_observer)
```

Comment:
would it make sense to set weight to `None`?  Not sure if there are cases where the right thing to do for current op is to reuse the weight observer of the previous op.

Reply:
we don't have None observers right now I think, maybe I can set it to a NoopObserver